scripts/test/lint.py

```diff
@@ -344,24 +376,20 @@ class UsesInvalidCategory(Lint):
         "chrome.extensions.vm",
     ]
 
-    root_path = os.path.abspath(os.path.join(__file__, "../../.."))
-    categories_txt = os.path.join(root_path, "categories.txt")
-    with open(categories_txt) as file:
-        CATEGORIES = [line.rstrip() for line in file]
-        logger.debug(CATEGORIES)
+    
 
-    name = "Uses an invalid category"
-    recommendation = f"Set $category to a category in {categories_txt} or exclude the package in the linter"
+    name = "Doesn't use the function VM-Get-Category"
+    recommendation = f"Set $category to retrieve the category from VM-Get-Category or exclude the package in the linter"
```

Comment:
I think it would be useful to be more explicit about how to set `$category`
```suggestion
    recommendation = f"Set '$category = VM-Get-Category($MyInvocation.MyCommand.Definition)' or exclude the package in the linter"
```